### DISCUSSION TITLE
How can I move randomly with gpiozero.Robot

### DISCUSSION THREAD

**Discussion by plumjungwoo** [gpiozero/gpiozero#1033] [Q&A] [answered]
I was working with gpiozero to make an OpenCV robot.

robot = gpiozero.Robot(right=(17,27), left=(5,6))
with this code, I can move the robot in four directions.

Is there a way to adjust the speed of each motor individually so that it moves diagonally or turns itself in a random direction?

**Comment by lurch** [comment, marked as answer]
Yes, have a look at the "curve" parameters to https://gpiozero.readthedocs.io/en/stable/api_boards.html#gpiozero.Robot.forward

**Reply by bennuttall** [reply]
Beat me to it :)

**Reply by plumjungwoo** [reply]
Thank you
